components/availability/src/Availability.svelte

```diff
@@ -646,6 +662,195 @@
     }
   }
   // #endregion Date Change
+
+  //#region dragging
```

Comment:
I think I'd like to see a lot of this in it's own JS file (and as mentioned it's own Open Source package too - I think marketing and talent acq would absolutely love that )